### ISSUE TITLE
#extension GL_ARB_shader_atomic_counter_ops meets compiler error.

### ISSUE THREAD

**Issue by ZhiqianXia** [KhronosGroup/glslang#2680]
Hi ,
    There is a compiler error about #extension **GL_ARB_shader_atomic_counter_ops**.

shader:
```
#version 450 core
#extension GL_ARB_shader_atomic_counters: enable
#extension GL_ARB_shader_atomic_counter_ops: enable
layout (binding=0) uniform atomic_uint counter;
layout (binding=1) uniform atomic_uint calls;
in highp vec2 inPosition;
out highp vec3 vsPosition;
out highp vec4 vsColor;
void main()
{
               gl_Position = vec4(inPosition, 0.0, 1.0);
               vsPosition = vec3(inPosition, 0.0);
               vec4 outColor = vec4(1.0);
               uint returned = atomicCounterAddARB(counter, 4u);
               uint after = atomicCounter(counter);
               if(after == returned) outColor = vec4(0.0f);
               atomicCounterIncrement(calls);
               vsColor = outColor;
}
```

**Comment by greg-lunarg** [comment]
Are you generating SPIR-V or just validating? If SPIR-V, are you generating for OpenGL or Vulkan?

**Comment by greg-lunarg** [comment]
GL_ARB_shader_atomic_counter_ops, along with GL_ARB_shader_atomic_counters, were merged into core at #version 460. This is probably why GL_ARB_shader_atomic_counter_ops was never added to the validator.

Technically, this is a valid shader, however at my funding level I will not be able to fix this in the near term. If someone wanted to go to the trouble of adding code to glslang to make this pass, I would consider allowing it.

**Comment by ZhiqianXia** [comment]
https://github.com/KhronosGroup/glslang/pull/2687

**Comment by greg-lunarg** [comment]
Fixed by #2687
